python/ray/worker.py

```diff
@@ -1304,7 +1302,8 @@ def connect(node,
     # that is not true of Redis pubsub clients. See the documentation at
     # https://github.com/andymccurdy/redis-py#thread-safety.
     worker.redis_client = node.create_redis_client()
-
+    _initialize_internal_kv(
```

Comment:
Can you save the GcsClient somewhere, so it can be reused for other purposes without creating another channel to GCS? And just to make sure, this is a no op until redis clients are replaced by internal kv calls?

Reply:
I don't think so. gcs client is introduced by me to reuse cpp gcs_client and it's for internal kv only. but anyway I can put it there.

Reply:
I mean I will probably either create a GCS pubsub client here, or add pubsub functions to GcsClient and use GcsClient for pubsub. I can do the refactoring later too.

Reply:
I'd prefer adding it to gcs client so that it looks all related to gcs. we can sync about pros and cons later.